cmd/CMakeLists.txt

```diff
@@ -14,11 +14,13 @@ function(add_cmd CMD_SRC IS_GUI)
         $<IF:$<BOOL:${IS_GUI}>,mrtrix::gui,mrtrix::headless>
         mrtrix::exec-version-lib
     )
-    set_target_properties(${CMD_NAME} PROPERTIES 
+    set_target_properties(${CMD_NAME} PROPERTIES
+        MACOSX_BUNDLE ${IS_GUI}
+        MACOSX_BUNDLE_ICON_FILE mrview.icns
         LINK_DEPENDS_NO_SHARED true
         RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin
     )
-    install(TARGETS ${CMD_NAME} RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+    install(TARGETS ${CMD_NAME} RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} BUNDLE DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
Can we indent this on multiple lines for better readability?

Reply:
Done